src/coreComponents/physicsSolvers/contact/SolidMechanicsAugmentedLagrangianContact.cpp

```diff
@@ -113,6 +116,12 @@ void SolidMechanicsAugmentedLagrangianContact::registerDataOnMesh( dataRepositor
         setRegisteringObjects( this->getName()).
         setDescription( "An array that holds the sliding tolerance." );
 
+      subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::dispJumpUpdPenaltyString() ).
```

Comment:
what's special about this? can't we use the `dispJump` we already have?

Reply:
This field is used to update the penalty. I need to keep track of the `dispJump` from the previous outer loop iteration. To update the penalty field, I compare `dispJump` with `dispJumpUpdPenalty`.